### PR TITLE
Add entity content and description.

### PR DESCRIPTION
**Pull request by kadams54**

### CODE REVIEW

**Comment by kadams54** [comment]
## QA

Open up the entity details. The description should now be displayed below the header.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3273/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3274/
Test PASSed.

**Comment by hatched** [comment]
QA Notes:
It only appears to work for the charms?

File: jujugui/static/gui/src/app/components/entity-content/entity-content.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+              {optionsList}
+            </dl>
+        </div>
+      );
+    },
+
+    render: function() {
+      var entity = this.props.entityModel.toEntity();
+
+      var description = entity.description ?
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-content/entity-content.js]: could you move these into separate functions?

File: jujugui/static/gui/src/app/components/entity-content/test-entity-content.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+'use strict';
+
+var juju = {components: {}};
+var testUtils = React.addons.TestUtils;
+
+chai.config.includeStack = true;
+chai.config.truncateThreshold = 0;
+
+describe('EntityContent', function() {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/entity-content/test-entity-content.js]: There is a lot of logic not being tested here, can you do the full output for each path.

**Comment by huwshimi** [comment]
I've taken this over. See https://github.com/juju/juju-gui/pull/913

**Pull request closed without merging**